app/src/main/java/org/greenstand/android/TreeTracker/fragments/NewTreeFragment.kt

```diff
@@ -103,6 +107,24 @@ class NewTreeFragment :
                     fragmentNewTreeDBH.text.toString())
             }
         }
+
+        fragmentNewTreeGPS.setOnClickListener {
+            lifecycleScope.launch {
+                progressBar.visibility = View.VISIBLE
+                progressBar.setOnClickListener {
+                    // Do nothing. We want to intercept taps on screen while loading is shown
```

Comment:
What would happen if we don't set the listener here?

Reply:
If we don't set the click listener than the view will not intercept clicks. This is covering the whole screen to avoid someone from doing something that can mess things up.